study.md

```diff
@@ -135,42 +135,43 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a breif description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+The rules for sending documents to and from on the internet.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client is and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
-```
+Client is the website, and a server is the host which accepts connections from Clients and allows users to.....use the client.  ```
 
 ## Describe the 4 most common HTTP verbs
 
 What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
-```
+Post - creates a new resouce within the parent - you would use it to, for example, post a new message to a message board.
+Get - Is the most common verb and is required by servers to be implemented.  you would use it to read files, or messages
+put - update or overwrite, you would use this reuest to, say replace a message
+and delete - delete, use this to delete items
 
 ## Describe what a Response is
 
 What is a response? what does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+After a server receives a request, it sends a response which contains a status line, the http version, the status code, and a message (optional).  404 would be a common one, means not found.   403 is also common, it means forbidden.  200, means OK.
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
-```
+You would specify what content you wanted to GET and use the command line?  I think,
+Curl is a little confusing in that answers were all very generalized on the web and the README was very dry.  But it looks like you specify which content you'd like to GET (example:  get the 10 most recent posts from a thread)
 
```

Comment:
I think you just overthought this one. 
Entering this in the CLI would output the html of reddit.com

`curl --request GET https://www.reddit.com`